src/gui/import_export.rs

```diff
@@ -0,0 +1,111 @@
+use crate::tungsten_halogen::reference_from_filament_temp;
+use crate::{ThreadId, ThreadResult};
+use egui::{Button, Context, Slider};
+
+pub struct ImportExportWindow {
+    tungsten_filament_temp: u16,
+}
+
+impl ImportExportWindow {
+    pub fn new() -> Self {
+        Self {
+            tungsten_filament_temp: 2800,
+        }
+    }
+
+    pub fn draw(
+        &mut self,
+        ctx: &Context,
+        config: &mut crate::config::SpectrometerConfig,
```

Comment:
I think it would be more ideal if a specific UI window did not depend on the *entire* program config. But it used so many fields that this was the more pragmatic approach. But I would like discussing how we could pass in dependencies more fine grained maybe. I don't know if the same will hold for the other windows. But this window depends on a lot of the config.

Reply:
I think we should leave it as it is for now. I hope when the restructuring is done it will be easier to find a sensible approach to split the config.

Reply:
I agree